app/Http/Controllers/EventController.php

```diff
@@ -99,6 +101,33 @@ public function join(Request $request, Event $event)
         return redirect()->route('profile.show')->banner('Inscrição realizada com sucesso!');
     }
 
+    public function remove(Request $request,Event $event, Enrollment $enrollment)
```

Comment:
You need authorization for this endpoint, or else any user can remove the enrollment of other users. I think there is a middleware to enforce that the calling user is an admin

Reply:
There is no middleware, but you can use something like this to achieve the same effect https://github.com/semanadeinformatica/website-24/blob/93ed09e7916c74aaa8e3397788a67169d5e42107/app/Http/Controllers/EventController.php#L37

Reply:
Yeah, I added a check for users that are admin, being logged in or not. I think its kinda dumb, but a good enough solution for now.